src/pal/src/thread/thread.cpp

```diff
@@ -2456,7 +2465,13 @@ PAL_GetStackLimit()
     
     pthread_t thread = pthread_self();
     
+#if HAVE_PTHREAD_ATTR_GET_NP
+    status = pthread_attr_get_np(thread, &attr);
+#elif HAVE_PTHREAD_GETATTR_NP
     status = pthread_getattr_np(thread, &attr);
+#else
+    #error Dont know how to get thread attributes on this platform!
```

Comment:
This #error isn't whitespace consistent with the above.  Please remove the prepending spaces.